Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -162,7 +162,7 @@ public int Sell(string symbol, decimal quantity)
         /// <seealso cref="Order(string, double)"/>
         public int Order(string symbol, double quantity)
         {
-            return Order(symbol, (int) quantity);
+            return Order(symbol, (int) Math.Abs(quantity) * -1);
```

Comment:
I think we need to revert this, we want the order function to respect the input quantity